web/src/SidebarKeyboardShortcuts.test.tsx

```diff
@@ -0,0 +1,58 @@
+import { MemoryRouter } from "react-router"
+import React from "react"
+import PathBuilder from "./PathBuilder"
+import { createMemoryHistory } from "history"
+import { mount } from "enzyme"
+import { twoResourceView } from "./testdata"
+import SidebarItem from "./SidebarItem"
+import SidebarKeyboardShortcuts from "./SidebarKeyboardShortcuts"
+
+const fakeHistory = createMemoryHistory()
+const pathBuilder = new PathBuilder("localhost", "/")
+const shortcuts = (items: SidebarItem[], selected: string) => {
+  let root = mount(
+    <MemoryRouter initialEntries={["/"]}>
+      <SidebarKeyboardShortcuts
+        items={items}
+        selected={selected}
+        history={fakeHistory}
+        pathBuilder={pathBuilder}
+      />
+    </MemoryRouter>
+  )
+  return root
+    .find(SidebarKeyboardShortcuts)
+    .instance() as SidebarKeyboardShortcuts
+}
+
+it("navigates forwards", () => {
```

Comment:
nice! worth a test that e.g. cmd+J/cmd+K don't have an effect? (i don't feel strongly)